### ISSUE TITLE
Remove api key from code?

### ISSUE THREAD

**Issue by Silex** [swflint/pushover.el#2]
Hello,

https://github.com/swflint/pushover.el/blob/master/pushover.el#L20

Are you sure you want a real key to be part of the code? If yes, simply close this issue.

**Comment by swflint** [comment]
Yep.  Less configuration for a user, and it already is set up with an image and the like.

**Comment by Silex** [comment]
Alright.

**Comment by swflint** [comment]
It seems you think I shouldn't, but what are your thoughts?

**Comment by Silex** [comment]
I don't know pushover, but it just feels wrong to have production settings in a repo... milkypostman had the same thought at https://github.com/melpa/melpa/pull/4029

**Comment by swflint** [comment]
Yeah, I'm probably going to remove it, but I'll probably do so with a `defcustom`.  Feel free to submit a pull request.
